docs/src/modules/components/Demo.js

```diff
@@ -295,7 +295,7 @@ const DemoRootJoy = joyStyled('div', {
     backgroundColor: alpha(grey[50], 0.2),
     ...theme.applyDarkStyles({
       borderColor: alpha(grey[700], 0.3),
-      backgroundColor: alpha(blueDark[800], 0.2),
+      backgroundColor: '#09090B',
```

Comment:
```suggestion
      backgroundColor: alpha(blueDark[800], 0.2),
```

I think the previous value here looks a bit better and clashes a bit less with the demos and overall docs! Now that the blacker dark mode has been merged, we don't need this one-off color 😬